docs/fundamentals/networking/http/httpclient-guidelines.md

```diff
@@ -52,6 +52,22 @@ To summarize recommended `HttpClient` use in terms of lifetime management, you s
 
 For more information about managing `HttpClient` lifetime with `IHttpClientFactory`, see [`IHttpClientFactory` guidelines](../../../core/extensions/httpclient-factory.md#httpclient-lifetime-management).
 
+## Resilience policies with static clients
+
+It is possible to configure a `static` or *singleton* client to use any number of resilience policies using the following pattern:
+
+```csharp
+var retryPolicy = HttpPolicyExtensions
```

Comment:
Where does the `HttpPolicyExtensions` type come from? We need both a NuGet package referenced here and a `using <namespace>`.

Reply:
The `HttpPolicyExtensions` comes from `Polly.Extensions.Http`